exo-deploy/package.json

```diff
@@ -4,20 +4,20 @@
   "author": "Hugo Ho",
   "dependencies": {
     "async": "2.1.4",
-    "aws-sdk": "2.7.10",
+    "aws-sdk": "2.7.27",
     "chalk": "1.1.3",
     "dashify": "0.2.2",
-    "fs-extra": "1.0.0",
-    "handlebars": "4.0.5",
-    "inquirer": "1.2.3",
+    "fs-extra": "2.0.0",
+    "handlebars": "4.0.6",
+    "inquirer": "3.0.1",
```

Comment:
I have been deploying from this branch and I haven't run into any problems. I also just checked the changelogs for `fs-extra` and `inquirer` and none of the breaking changes affect us.